hack/build.sh

```diff
@@ -50,7 +50,11 @@ function squash {
 dirs=${VERSION:-$VERSIONS}
 
 for dir in ${dirs}; do
-  IMAGE_NAME="${BASE_IMAGE_NAME}-${dir//./}-${OS}"
+  if [ "$dir" == "2.6" ]; then
+    IMAGE_NAME=rhscl/rh-mongodb${dir//./}-${OS}
```

Comment:
so we already raised an objection for that naming convention, since this naming scheme will break down if we ever want to support a repo with a name that ends in a number.